modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -3335,8 +3333,6 @@ definition {
 				actionActive = "Yes",
 				actionLabel = "Action Label",
 				status = "Success");
-
-			AssertConsoleTextPresent(value1 = "Hello world");
```

Comment:
@andreefarias why are we removing the assertion?

Reply:
This automation test never passed locally or on CI, only manually. 

I investigated and saw that looks like the macro doesn't assert in the right console level, but I think that this step is not necessary since the status is moved to success before. What do you think, makes sense?

cc: @patriciajeanperez

Reply:
ooh I wonder if `WaitForConsoleTextPresent` works if you really want to assert the console log

Reply:
I tried, but not work. 😢 

Some months ago, I and @davisaints try some different ways, but no success. I think is necessary for a different configuration to assert the correct log, but I don't know.

cc: @patriciajeanperez

Reply:
@andreefarias there isn't much value in this test if we're removing the actual assertion since we can't actually confirm that the action was triggered. We would either need to investigate further into why `print` isn't working while running our Poshi tests or find a different way to assert the action is working.